src/main/web_src/src/pages/gruppe/PersonVisning/PersonVisning.less

```diff
@@ -13,7 +13,7 @@
 	}
 	&_actions {
 		display: flex;
-		justify-content: flex-end;
+		justify-content: center; // flex-start her isteden?
```

Comment:
Synes center så best ut, men blir kanskje rart når de andre knappene er venstrestilt i gruppeoversikten ovenfor 😜

Reply:
Helt enig! (Måtte ta en liten titt på hvordan det så ut med flex-start, og det ble bare merkelig 😅  )
Men jeg kunne tenkt meg litt mer space over og under for å få dem bedre fram. F.eks. `margin: 5px 0 10px 0;` 😄

Reply:
Ja ikkseant 😂
Se der ja, det ble mye luftigere, takk for tipset! 🤩